svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -272,9 +271,9 @@ def populate_rlz_cbx(self):
         #         self.hdata = self.npz_file['loss_maps-rlzs']
         #     _, n_rlzs = self.hdata.shape
         #     self.rlzs = [str(i+1) for i in range(n_rlzs)]
+        self.rlzs = [key for key in self.npz_file.keys() if key != 'imtls']
         self.rlz_cbx.clear()
```

Comment:
I am not sure if the call to .keys() is needed. I would replace it with a `sorted(self.npz_file)`.